backend/app/Http/Controllers/StudentController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Student;
+use Illuminate\Http\Request;
+
+class StudentController extends Controller
+{
+  // register a new user
+  public function store(Request $request) {
```

Comment:
Also we use as our standard for PHP functions and classess these format for the `{ }` blocks.
```
public function Test()
{
    // content here
}
```

Reply:
Got it Sir. I'll work on it.